### PR TITLE
STIJ-32: Refactored header organism and added mobile support.

### PR DESCRIPTION
**Pull request by Jeroen005**

### CODE REVIEW

File: components/31-molecules/building-blocks/mijn-gent-block/_mijn-gent-block.scss [hunk 1]
```diff
@@ -6,35 +6,54 @@
...
   }
 
-  .login-link i {
-    margin-left: .625rem;
+  .login-link {
+    display: inline-block;
+    position: relative;
+    height: 2rem;
+
+    @include tablet() {
```
Comment (meirege) [inline components/31-molecules/building-blocks/mijn-gent-block/_mijn-gent-block.scss]: I always use this mixin like this:
`@include tablet { ... }`

Perhaps we should agree on 1 way of writing it?

File: components/31-molecules/logo/_logo.scss [hunk 2]
```diff
@@ -1,17 +1,49 @@
...
+        font-weight: bold;
+        line-height: 1rem;
+
+        &::after {
+          position: absolute;
+          top: 1rem;
+          right: 0;
+          width: 33%;
+          height: calc(100% - 2rem);
+          padding-left: 5px;
```
Comment (meirege) [inline components/31-molecules/logo/_logo.scss]: rem?

File: components/31-molecules/logo/logo.twig [hunk 3]
```diff
@@ -3,6 +3,8 @@
       {% include '@image' with { 'src': '/styleguide/img/svg/logo.svg', 'alt_text': 'Home' } %}
     </a>
     <div class="site-name">
-      {% include '@paragraph' with { 'text': 'Stad Gent Sitename' } %}
+        <div>
```
Comment (meirege) [inline components/31-molecules/logo/logo.twig]: Why the extra div?

Comment (Jeroen005) [reply components/31-molecules/logo/logo.twig]: An extra wrapper div is needed to align the site title in the middle of the header.

**Pull request merged**